adapter/internal/oasparser/model/resource.go

```diff
@@ -88,6 +91,42 @@ func (resource *Resource) GetMethodList() []string {
 	return methodList
 }
 
+// GetRewriteResource returns the rewrite upstream path for a given resource.
+func (resource *Resource) GetRewriteResource() (string, bool) {
+	rewritePath := ""
+	rewriteMethod := false
+	pathOrder := 0
+	for _, method := range resource.methods {
+		if len(method.policies.In) > 0 {
+			for _, policy := range method.policies.In {
```

Comment:
any reason for checking this condition explicitly `if len(method.policies.In) > 0`

Reply:
+1 to remove it